.github/workflows/release-please.yml

```
@@ -52,23 +52,24 @@ jobs:
           token: ${{ steps.otelbot-token.outputs.token }}
           fetch-depth: 0
 
-      # previous commiter does not have CLA signed, and the commit is missing updates to package-lock.json,
-      # so we re-write that commit here with the proper changes
+      # If we've created/updated a PR:
+      # 1. Amend the author to 'otelbot', which passes EasyCLA.
+      # 2. Sync the legacy "dependencies" key in package-lock.json.
       - name: Update package-lock.json in PR
-        # only update if a PR has been created
         if: ${{ steps.release.outputs.pr }}
         run: |
-          npm install --ignore-scripts --package-lock-only
-          git add package-lock.json
           git config user.name otelbot
           git config user.email 197425009+otelbot@users.noreply.github.com
-          git commit --amend --reset-author --no-edit
-          git push --force
+          git rebase main -x "git commit --amend --reset-author --no-edit"
+          npm install --ignore-scripts --package-lock-only
+          git add package-lock.json
+          git commit -m "chore: sync package-lock.json 'dependencies' key"
+          git push
```

since we still re-write the history.
```suggestion
          git push --force
```